web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -916,20 +916,24 @@ fixmystreet.update_report_a_problem_btn = function() {
     var visible = true;
     var text = translation_strings.report_a_problem_btn.default;
 
+    var new_report_url = function(lonlat) {
```

Comment:
Why isn't this `var new_report_url = '/report/new?longitude=' + center.lon + '&latitude=' + center.lat;`? :)